content/hi/docs/concepts/architecture/cgroups.md

```diff
@@ -0,0 +1,125 @@
+---
+title: सीग्रुप v2 के बारे में
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+Linux पर, {{< glossary_tooltip text="control groups" term_id="cgroup" >}}
+प्रक्रियाओं को आवंटित संसाधनों को सीमित करें।
+
+{{< Glosary_tooltip text="kubelet" term_id="kubelet" >}} और
```

Comment:
This causes [Netlify build error](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/63f279f8c0a0b40008fbcd92#L127).
> template for shortcode "Glosary_tooltip" not found

You should use the shortcode `glossary_tooltip` as follows:

```suggestion
{{< glossary_tooltip text="kubelet" term_id="kubelet" >}} और
```